src/ApiFunction.ts

```diff
@@ -80,12 +76,6 @@ export class ApiFunction extends Construct {
     });
     alarm.applyRemovalPolicy(RemovalPolicy.DESTROY);
 
-    const subscriptionFilter = new SubscriptionFilter(this, 'error-logs-subscription', {
```

Comment:
Wordt deze filter nu wel ergens anders opgevangen / gemonitored?

Reply:
De errors worden nu geteld in een custom metric, op die metric zit een alarm. De errors zelf zijn in de logs en met de insights-query die in de normale monitoring zit opgepakt, zodat we bijzondere errors niet gaan missen.